bigchaindb/common/exceptions.py

```diff
@@ -22,7 +22,11 @@ class DoubleSpend(Exception):
     """Raised if a double spend is found"""
 
 
-class InvalidHash(Exception):
+class ValidationError(Exception):
+    """Raised if any error with object schema"""
```

Comment:
I would also rename this exception to something more precise, e.g. `TxSchemaValidationError`. (There are many ways an error might arise when validating a transaction; an error in validating its JSON structure against the JSON schema is only a subset.)